SimTracker/TrackHistory/python/TrackQuality_cff.py

```diff
@@ -11,6 +11,8 @@
 		associatePixel = cms.bool(True),
 		associateStrip = cms.bool(True),
 		associateRecoTracks = cms.bool(True),
-		ROUList = copy.deepcopy(tabh.trackAssociatorByHits.ROUList)
+		ROUList = copy.deepcopy(tabh.trackAssociatorByHits.ROUList),
+                pixelSimLinkSrc = cms.InputTag("simSiPixelDigis"),
```

Comment:
this is from merging cms-btv-pog:FixTrackQuality-from-CMSSW_7_6_1. @imarches is this required for 80X? What is the usual procedure in case of dependent merges?